src/adapters/genk-staking.adapter.ts

```diff
@@ -0,0 +1,48 @@
+import {BindingScope, extensionFor, injectable} from '@loopback/core';
+import {BigNumber} from 'ethers';
+import {STAKING_ADAPTERS_EXTENSION_POINT} from '../keys';
+import {BaseStakingContractAdapter, StakingAsset} from '../staking';
+import {GenkStaking__factory} from '../types';
```

Comment:
Please use the full path to import `GenkStaking__factory`.

Reply:
Sure thing! :+1: done